src/common/rtweekend.h

```diff
@@ -12,8 +12,14 @@
 #include <cstdlib>
 #include <limits>
 #include <cmath>
+#include <memory>
 
 
+// Usings
+
+using std::shared_ptr;
```

Comment:
These `using`s need to be explicitly written out in the text. Otherwise, everyone is going to be asking why `shared_ptr` and `make_shared` is undefined

Reply:
Worse than that, `hittable_list` is presented just before the `rtweekend.h` common header. Will rework.